opennebula/resource_opennebula_virtual_network.go

```diff
@@ -300,6 +241,86 @@ func resourceOpennebulaVirtualNetwork() *schema.Resource {
 	}
 }
 
+func ARFields() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"id": {
+			Type:     schema.TypeString,
+			Computed: true,
+		},
+		"ar_type": {
+			Type:        schema.TypeString,
+			Optional:    true,
+			Default:     "IP4",
+			Description: "Type of the Address Range: IP4, IP6. Default is 'IP4'",
+			ValidateFunc: func(v interface{}, k string) (ws []string, errors []error) {
+				validtypes := []string{"IP4", "IP6", "IP6_STATIC", "IP4_6", "IP4_6_STATIC", "ETHER"}
+				value := v.(string)
+
+				if inArray(value, validtypes) < 0 {
+					errors = append(errors, fmt.Errorf("Address Range type %q must be one of: %s", k, strings.Join(validtypes, ",")))
+				}
+
+				return
+			},
+		},
+		"ip4": {
```

Comment:
`computed_ip4` is missing

Reply:
I may be wrong but I did this because in [5.12 One doc](https://docs.opennebula.io/5.12/operation/references/vnet_template.html#the-address-range), for ipv4 address ranges the ip4 address is mandatory (so ip4 would be equal to computed_ipv4 in all cases), however it seems that it's not always the case for ip6.

However I understand that this seems weird so I can update this

Reply:
OK so not required here.